migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ContentObjectTransformer.java

```diff
@@ -381,18 +382,20 @@ private void transformPremis(PID bxc3Pid, PID bxc5Pid) {
         premisTransformer.compute();
 
         // Add migration event
-        addMigrationEvent(premisLogger);
+        addMigrationEvent(premisLogger, depositBxc3.getFullname());
```

Comment:
this event describes the migration of the object from boxc3 to 5, so it should have the migration-util 5.0 agent.